docs/helpers.rst

```diff
@@ -38,14 +38,22 @@ on what marks are and for notes on using_ them.
      uses. When ``transaction=True``, the behavior will be the same as
      `django.test.TransactionTestCase`_
 
+   :type reset_sequences: bool
+   :param reset_sequences:
+     The ``reset_sequences`` argument will ask to reset auto increment sequence
+     values (e.g. primary keys) before running the test.  Defaults to
+     ``False``.  Must be used together with ``transaction=True`` to have an
```

Comment:
> Must be used together with ``transaction=True``

Do you think this should get done automatically, i.e. using `reset_sequences` would pull in / use the transactional_db fixture automatically?